ui/page/dexclient/markets.go

```diff
@@ -0,0 +1,247 @@
+package dexclient
+
+import (
+	"context"
+	"fmt"
+
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/client/db"
+	"gioui.org/layout"
+
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/modal"
+	"github.com/planetdecred/godcr/ui/page/components"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+type (
+	C = layout.Context
+	D = layout.Dimensions
+)
+
+const MarketPageID = "Markets"
+
+type Page struct {
+	*load.Load
+	ctx                     context.Context
+	ctxCancel               context.CancelFunc
+	login                   decredmaterial.Button
+	initialize              decredmaterial.Button
+	addDex                  decredmaterial.Button
+	sync                    decredmaterial.Button
+	isConnectedToDcrNetwork bool
```

Comment:
Shouldn't need this. Use `pg.WL.MultiWallet.IsConnectedToDecredNetwork()` instead where it's needed.